addons/account/models/account_move_line.py

```diff
@@ -1541,6 +1541,26 @@ def copy_data(self, default=None):
                 line._copy_data_extend_business_fields(values)
         return data_list
 
+    def _search_panel_domain_image(self, field_name, domain, set_count=False, limit=False):
```

Comment:
@Polymorphe57 Is it better to do this here in business code, or should we do something similar generally when `set_count=False`?

I'm not sure if it has any impact on non indexed `field_name`, but it has a huge impact on those who are.

Reply:
After testing, it looks like this is even worse :facepalm: 

We will simply filter on the roots of the accounts of the company instead of on the domain of the list view.

So no need for your input anymore, thanks anyways!